### ISSUE TITLE
Various terms missing for recurrent respiratory papillomatosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5553]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6754618/

- laryngitis
- palmar warts
-plantar warts
-recurrent growth of papillomas in the upper airway

**Comment by mgriese** [comment]
Oh, this is a very difficult field,.....

Preferred term label:
Subglottic laryngitis

Synonyms
Subglottic laryngitis, pseudocroup, croup, laryngitis, viral croup, spasmodic croup 

(all these can  be new terms with the same definition)

Definition (free text, please give PubMed ID)
Narrowing of the larynx, commonly occuring during viral respiratory tract infections, in particular in children, leads to symptoms such as hoarseness, a ‘barking’ cough, stridor, and sometimes dyspnea and respiratory failure. 

PMID: 31680234

Parent term (use hpo.jax.org/app)
Upper airway obstruction HP:0002781

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Viral airway infections
Diptheria

Your nano-attribution (ORCID)
LMU: crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
(syn: also just laryngitis)

**Comment by pnrobinson** [comment]
Laryngeal papillomatosis, syn. Laryngeal papillomas, syn., Multiple laryngeal papillomas

Glottis papillomas

Bronchial papillomatosis

Tracheal papillomatosis
